lib/ocpp/v201/charge_point.cpp

```diff
@@ -573,6 +573,7 @@ AuthorizeResponse ChargePoint::validate_token(const IdToken id_token, const std:
     response = this->authorize_req(id_token, certificate, ocsp_request_data);
 
     if (auth_cache_enabled) {
+        update_id_token_cache_lifetime(response.idTokenInfo);
```

Comment:
nit: just to be consistent. We are using `this` everywhere
```suggestion
        this->update_id_token_cache_lifetime(response.idTokenInfo);
```

Reply:
Yep, good call, missed that one. Please keep these nit's coming ;)